src/Expr.h

```diff
@@ -368,6 +368,9 @@ enum class MemoryType {
      * "local" in OpenCL, and "threadgroup" in metal. Can be shared
      * across GPU threads within the same block. */
     GPUShared,
+
+    /**Allocating and Locking Cached Memory to act as local memory(TCM)*/
```

Comment:
/**Allocate locked cached memory to act as local memory. */

Reply:
Modified the comment